index.css

```diff
@@ -2,23 +2,71 @@ body {
   background-color: #fcebb6;
   color: #221811;
   font-family: Verdana, Geneva, Tahoma, sans-serif;
-  width: 80%;
+  width: 70%;
   margin-right: auto;
   margin-left: auto;
 }
 header {
   background-color: #d2b48c;
-  text-align: center;
+  height: 150px;
 }
 header h1{
-  line-height: 200%;
+  padding-top: 45px;
+  padding-left: 220px;
+  font-size: 3em;
 }
+
 nav{
   text-align: center;
   font-weight: bold;
+  font-size: 1.5em;
+  padding-top: 10px;
+}
+nav a{
```

Comment:
ĐỌc css lint fix lại toàn bộ style file css nhé em https://github.com/CSSLint/csslint/wiki/Rules